### PR TITLE
Matter FVP persistent memory via files

### PR DESCRIPTION
**Pull request by ATmobica**
Provides the persistent memory ot the Matter examples run on FVP.
Use FVP command-line options to support persistent memory via files.

### CODE REVIEW

File: docs/guides/openiotsdk_platform_overview.md [hunk 1]
```diff
@@ -68,17 +68,41 @@ Configuration of Mbed TLS is in
...
+
+More details about `FVP command-line options` you can find
+[here](https://developer.arm.com/documentation/100966/1116/Getting-Started-with-Fixed-Virtual-Platforms/FVP-command-line-options).
+
+You have to be very careful when choosing memory parameters. Use the
+`--list-memory` option to see the list of instances and memory spaces for your
+`FVP` model.
+
+Depending on your application, choose the right memory instance, memory space,
+address and size.
```
Comment (pan-) [inline docs/guides/openiotsdk_platform_overview.md:104]:
```suggestion
Two command lines options can be used to achieve persistence of a specific memory regions:
- `--dump` store the content of a memory region into a file when the model ends its execution
- `--data` load the content of a file into a specific memory region at startup

Use the `--list-memory` flag to see the list of instances and memory spaces for your
`FVP` model.

Visit the [FVP command line documentation](https://developer.arm.com/documentation/100966/1116/Getting-Started-with-Fixed-Virtual-Platforms/FVP-command-line-options) for more details about these flags.

Depending on your application, choose the right memory instance, memory space,
address and size.
```

File: docs/examples/openiotsdk_examples.md [hunk 2]
```diff
@@ -310,6 +326,42 @@ to `TFM_NS_APP_VERSION`.
...
+available
+[here](../guides/openiotsdk_platform_overview.md#fast-model-persistent-memory-via-files).
+Depending on the storage implementation, different flags are used in the `FVP`
+options.
+
+For block device storage use:
+
+```
+--dump mps3_board.sram=<file-path>@0:0x0,0x100000
+--data mps3_board.sram=<file-path>:0x0,0x100000
```
Comment (paul-szczepanek-arm) [inline docs/examples/openiotsdk_examples.md:350]: missing @0 ?

File: docs/examples/openiotsdk_examples.md [hunk 3]
```diff
@@ -310,6 +326,42 @@ to `TFM_NS_APP_VERSION`.
...
+
+```
+--dump mps3_board.qspi_sram=<file-path>@0:0x660000,0x12000
+--data mps3_board.qspi_sram=<file-path>@0:0x660000,0x12000
+```
+
+_Note_: The `file-path` must exist to use the `--data` option.
+
+[Open IoT SDK build script](../../scripts/examples/openiotsdk_example.sh)
+provides the `-K,--kvsfile` option to use the persistence options listed above.
```
Comment (paul-szczepanek-arm) [inline docs/examples/openiotsdk_examples.md:363]: is it upper case `K` or lower case `k`? It was lower case above

Comment (paul-szczepanek-arm) [reply docs/examples/openiotsdk_examples.md:363]: it's file, sorry

**Review by pan- — APPROVED** [review]

**Pull request merged**